decompiler/IR2/OpenGoalMapping.cpp

```diff
@@ -11,13 +11,12 @@ typedef OpenGOALAsm::InstructionModifiers MOD;
 const std::map<InstructionKind, OpenGOALAsm::Function> MIPS_ASM_TO_OPEN_GOAL_FUNCS = {
     // ----- EE -------
     // Instructions that are nopped
-    //    {InstructionKind::MTC0,
-    //     {".nop", {MOD::SKIP_IT}}},  // they only use this for performance counters / Count /
-    //     Debug
-    //    {InstructionKind::MTPC, {".nop", {MOD::SKIP_IT}}},
-    //    {InstructionKind::MFPC, {".nop", {MOD::SKIP_IT}}},
-    //    {InstructionKind::SYNCP, {".nop", {MOD::SKIP_IT}}},
-    //    {InstructionKind::SYNCL, {".nop", {MOD::SKIP_IT}}},
+    // they only use this for performance counters / Count / Debug
+    {InstructionKind::MTC0, {".nop", {MOD::SKIP_IT}}},
+    {InstructionKind::MTPC, {".nop", {MOD::SKIP_IT}}},
+    {InstructionKind::MFPC, {".nop", {MOD::SKIP_IT}}},
+    {InstructionKind::SYNCP, {".nop", {MOD::SKIP_IT}}},
+    {InstructionKind::SYNCL, {".nop", {MOD::SKIP_IT}}},
```

Comment:
specifically calling out this change, it allows you to decompile function that have small amounts of asm (perf counter stuff) without having to manually rewrite/edit the output.  `spatial-hash` has a ton of this